src/test/java/org/opensearch/searchrelevance/action/scheduledJob/ScheduledJobsIT.java

```diff
@@ -85,11 +87,28 @@ public void testMainActions_whenCreateReadDeleteScheduledExperiment_thenSuccessf
         assertNotNull(scheduledExperimentSource.get("schedule"));
         assertEquals(experimentId, scheduledExperimentSource.get("id"));
 
+        String getScheduledExperimentJobRunByIdUrl = String.join("/", SCHEDULED_JOBS_INDEX, "_doc", experimentId);
+
         // Here we have to wait until at last one
```

Comment:
Can we refine this comment?   It's a bit unclear to me...  And or, do we even need it?